### DISCUSSION TITLE
Plugin Textview preferences

### DISCUSSION THREAD

**Discussion by rui-nibau** [zim-desktop-wiki/zim-desktop-wiki#2576] [Show and tell]
Hello everybody.

Just an announcement for the release of the new version of my zim 0plugin, textview prefs 2.0.2, plugin that could be useful to other zim users.

you can define paddings around text, max width for text, background and text color.

• website: https://omacronides.com/projets/zim-plugin-textview-prefs
• source code: https://framagit.org/rui.nibau/zim-plugin-textview-prefs

![2 0-config](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/31863240/b5dc99b8-6395-420b-8cad-e526133e065d)

**Comment by rui-nibau** [comment]
**New version textview prefs 2.0.3**

Fix: [Issue 1](https://framagit.org/rui.nibau/zim-plugin-textview-prefs/-/issues/1)

• website / documentation / screenshots : https://omacronides.com/projets/zim-plugin-textview-prefs
• source code / issues : https://framagit.org/rui.nibau/zim-plugin-textview-prefs

**Reply by deminimis** [reply]
Thanks a lot for this. Any chance it could be implemented for new windows (like when you right click a note and open in new window)?

**Reply by rui-nibau** [reply]
I'll try to see what i can do.

Opened ticket: https://framagit.org/rui.nibau/zim-plugin-textview-prefs/-/issues/2

**Comment by rui-nibau** [comment]
### New version textview prefs 2.0.4

Fix: [Issue 2](https://framagit.org/rui.nibau/zim-plugin-textview-prefs/-/issues/2) : Apply to new windows

• website / documentation / screenshots / download : https://omacronides.com/projets/zim-plugin-textview-prefs
• source code / issues : https://framagit.org/rui.nibau/zim-plugin-textview-prefs

**Comment by sr79368142** [comment]
dear @rui-nibau ,Great plugin! on the windows platform zim preset Distraction Free Editing plugin settings are not perfect, full screen can not automatically line feed, ctrl + space shortcut does not take effect, I hope that Textview Preferences can complement these features and add the hidden menu bar, you can press ALT to call out the menu bar on options.
